src/binomial.jl

```diff
@@ -0,0 +1,98 @@
+#sfe(x) = log(n!*e^n/((n^n)*sqrt(2*pi*n))), for stirlerr
+const sfe = Float64[log(factorial(n)) - log(sqrt(2pi * n) * (n/Base.e)^n) for n in big(1):big(20)]
+# stirlerr(x) = log(x!) -log(sqrt(2*pi*x)*(n/e)^n)
+function stirlerr(n::Float64)
+    if (n <= 15)
+        @inbounds return(sfe[Int64(n)])
+    end
+    return lstirling_asym(n)
+end
+

```

Comment:
I think it should be `lstirling_asym(n + 1)`.  The `lstirling_asym` function returns the error from Stirling's approximation to `lgamma`, not `log(factorial(n)`

Reply:
I'm afraid there seems to be an error here. The `lstirling_asym()` is returning the error to the `ln(factorial(n))` instead of `ln(gamma(n))`. 
`f(x) = log(factorial(x)) - x*log(x) + x - 0.5*log(2*pi*x)`
`f(10) = 0.008330563433359472`
`lstirling_asym(10) = 0.00833056343336287`
`lstirling_asym(11) = 0.00757367548795184`